### PR TITLE
<Image/> -  border radius prop

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by roibWix** [comment]
@EgoziE waiting for <image/> show border prop to merge. need to rebase for the examples.js file

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6384

File: src/Image/Image.js [hunk 1]
```diff
@@ -37,12 +38,17 @@ const Image = ({
       className={st(classes.root, { tiled: isTiled }, className)}
       data-hook={dataHook}
       src={source}
-      style={{ ...style, ...imgStyle }}
+      style={{ ...style, ...imgStyle, ...{ borderRadius } }}
```
Comment (EgoziE) [inline src/Image/Image.js:41]: I believe no spread is needed:

```suggestion
      style={{ ...style, ...imgStyle, borderRadius }}
```

File: src/Image/Image.js [hunk 2]
```diff
@@ -78,6 +84,7 @@ Image.defaultProps = {
   width: '100%',
   fit: 'cover',
   position: 'center',
+  borderRadius: '8px',
```
Comment (EgoziE) [inline src/Image/Image.js:87]: This can be removed, because you already get the default value from the CSS file

File: src/Image/index.d.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 export type ImageFit = 'contain' | 'cover' | 'tile' | 'none';
 
 export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  borderRadius? : string,
```
Comment (EgoziE) [inline src/Image/index.d.ts:6]: If the type is pure CSS property, we use the direct type for it:

```suggestion
  borderRadius?: React.CSSProperties['borderRadius'];
```

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Image/Image.js (hunk 1 above)
Comment (roibWix) [reply src/Image/Image.js:41]: done

File: src/Image/Image.js (hunk 2 above)
Comment (roibWix) [reply src/Image/Image.js:87]: @EgoziE we don't want to keep it for the table of the API section? (default value)

File: src/Image/index.d.ts (hunk 3 above)
Comment (roibWix) [reply src/Image/index.d.ts:6]: done

**Review by EgoziE — APPROVED** [review]

**Review by milkyfruit — APPROVED** [review]
@roibWix PR looks great!

**Pull request merged**